features/placeholder.yml

```
@@ -0,0 +1,7 @@
+name: ::placeholder
+description: The `::placeholder` CSS pseudo-element selects text in `<input>` and `<textarea>` elements when no value is set.
```

Random idea: hint at the use case here?

```suggestion
description: The `::placeholder` CSS pseudo-element selects help text in `<input>` and `<textarea>` elements when no value is set.
```